app/scripts/templates/settings/device_name.mustache

```diff
@@ -0,0 +1,28 @@
+<div id="device-name" class="settings-unit">
+  <div class="settings-unit-stub">
+    <header class="settings-unit-summary">
+      <h2 class="settings-unit-title">{{#t}}Device name{{/t}}</h2>
+    </header>
+    <button class="settings-button secondary settings-unit-toggle hidden"
+      data-href="settings/device_name">
+      {{#t}}Change{{/t}}
+    </button>
+  </div>
+
+  <div class="settings-unit-details">
+    <div class="error"></div>
+
+    <form novalidate>
+      <p>
+      {{#t}}Device name{{/t}}
```

Comment:
I should probably remove the `device_name` template and view from this PR and file a separate PR when we are ready for that feature.